### ISSUE TITLE
Log of "Starting sessionMiddleware..." is forgotten debug log that needs removed

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#3615]
### What do you want and why?

This line needs removed: https://github.com/blitz-js/blitz/blob/main/packages/blitz-auth/src/server/auth-plugin.ts#L108

**Comment by Himanxu1** [comment]
I want to work on this issue @flybayer 
Assign this to me

**Comment by flybayer** [comment]
@Himanxu1 all yours! :)

**Comment by beerose** [comment]
Hey @Himanxu1 — technically, @noxify was the first to create a PR for this issue, so I'm going to merge @noxify's PR and close yours. 
For the future — before starting work, you can take a look at the linked PRs, previous discussions etc., to see if someone already covered an issue.

**Comment by noxify** [comment]
next time I will post a reply that I want to work on an issue - sorry for the confusion

**Comment by Himanxu1** [comment]
Okay @beerose
